### PR TITLE
Sign-in page 500 error message does not appear in toastr (+ messages localization)

### PR DESCRIPTION
**Pull request by antonAce**
Moved all server 4xx and 5xx code notifications from form error messages to toastr

### CODE REVIEW

File: Src/Campus.Master.API/UI/src/assets/locale/en.json [hunk 1]
```diff
@@ -64,6 +64,15 @@
             "SIGN-IN": "Sign in",
             "NOT-REGISTERED": "Not registered yet?",
             "SIGN-UP": "Sign Up"
+        },
+        "ERROR-TOASTR": {
+            "HEADER": "Error",
+            "NOT-AUTHORIZED": "You are not authorized",
+            "SERVER-ERROR": "A server error has occurred",
+            "MATCH": "Passwords don't match",
```
Comment (profileEnumerable) [inline Src/Campus.Master.API/UI/src/assets/locale/en.json:72]: Remove MATCH key value pair

File: Src/Campus.Master.API/UI/src/assets/locale/ua.json [hunk 2]
```diff
@@ -64,6 +64,11 @@
             "SIGN-IN": "Вхід",
             "NOT-REGISTERED": "Ще не зареєстровані?",
             "SIGN-UP": "Зареєструватися"
+        },
+        "ERROR-TOASTR": {
+            "HEADER": "Помилка",
```
Comment (profileEnumerable) [inline Src/Campus.Master.API/UI/src/assets/locale/ua.json:69]: Add Ukrainian localization for custom messages

**Review by profileEnumerable — COMMENTED** [review]
Change redirect route at the registration component

File: Src/Campus.Master.API/UI/src/app/auth/registration/registration.component.ts [hunk 3]
```diff
@@ -22,13 +24,24 @@ export class RegistrationComponent implements OnInit, OnDestroy {
...
   param = { minlength: 8, maxlength: 100 };
 
   private registerUser$: Subscription = new Subscription();
 
+  private toastStyles = {
+    toastClass: 'ngx-toastr server-error-toastr'
+  };
+
+  private responseLocaleMap = {
+    'Passwords don\'t match.': 'AUTH.ERROR-TOASTR.MATCH',
```
[the remark this answers is not in the document]
Comment (antonAce) [reply Src/Campus.Master.API/UI/src/app/auth/registration/registration.component.ts]: Done

File: Src/Campus.Master.API/UI/src/assets/locale/en.json (hunk 1 above)
Comment (antonAce) [reply Src/Campus.Master.API/UI/src/assets/locale/en.json]: Done

File: Src/Campus.Master.API/UI/src/assets/locale/ua.json (hunk 2 above)
Comment (antonAce) [reply Src/Campus.Master.API/UI/src/assets/locale/ua.json:69]: Done

**Review by profileEnumerable — APPROVED** [review]
Good job mate !

**Pull request merged**